src/components/notification-list-item.component.js

```diff
@@ -12,41 +13,48 @@ type Props = {
   navigationAction: Function,
 };
 
-const styles = StyleSheet.create({
-  container: {
-    borderBottomColor: colors.greyLight,
-    borderBottomWidth: 1,
-  },
-  wrapper: {
-    padding: 10,
-    flexDirection: 'row',
-  },
-  notificationInfo: {
-    flex: 1,
-    flexDirection: 'row',
-    alignItems: 'center',
-  },
-  avatar: {
-    backgroundColor: colors.greyLight,
-    borderRadius: 17,
-    width: 34,
-    height: 34,
-  },
-  titleContainer: {
-    flex: 1,
-  },
-  title: {
-    color: colors.black,
-    ...fonts.fontPrimary,
-    fontSize: normalize(12),
-    marginLeft: 10,
-  },
-  iconContainer: {
-    flex: 0.15,
-    alignItems: 'flex-end',
-    justifyContent: 'center',
-  },
-});
+const NotificationListItemContainer = styled.View`
+  border-bottom-color: ${colors.greyLight};
+  border-bottom-width: 1px;
+`;
+
+const Wrapper = styled.View`
+  flex-direction: row;
+  padding: 10px;
+`;
+
+const TitleComponent = styled(({ tag, children, ...props }) =>
+  React.createElement(tag, props, children)
+)`
+  flex: 1;
+  flex-direction: row;
+  align-items: center;
+`;
+
+const TitleContainer = styled.View`
+  flex: 1;
+`;
+
+const Title = styled.Text`
+  color: ${colors.black};
+  ${{ ...fonts.fontPrimary }};
```

Comment:
Please try to use `font-family: ${styledFonts.fontPrimary}`, while `styledFonts` is imported from `config`.